src/shared/services/form/form-validators.service.spec.ts

```diff
@@ -16,6 +16,16 @@ describe('FormValidatorsService', () => {
     expect(result.invalid).toBeFalsy();
   });
 
```

Comment:
please add the following test: 
field mandatory, BUT, with a regex that allows all whitespaces 
what happens in that case?
we should have a validation error, because the field is mandatory. Even if the regex in theory allows it